lib/auth/strategy.js

```diff
@@ -92,7 +92,8 @@ Strategy.prototype.userProfile = function (accessToken, done) {
                 image: userProfile.avatar,
                 name: userProfile.name,
                 userId: userProfile.id,
-                role: RoleNames[userTeamRole.role] || ''
+                role: RoleNames[userTeamRole.role] || '',
+                SSOGroups: userProfile.SSOGroups || []
```

Comment:
```suggestion
                SSOGroups: userProfile.SSOGroups || undefined
```
Suggestion to default to undefined (so it won't appear on the object) rather than an empty array. This will mean the property won't appear for non-SSO users (the majority)